android/app/src/main/gen/com/example/learningcompassexp/BuildConfig.java

```diff
@@ -0,0 +1,8 @@
+/*___Generated_by_IDEA___*/
```

Comment:
Don't know how these 3 first files were suddenly generated, but I think you should add them to .gitignore since they seem to only concern your IDE.